### ISSUE TITLE
Wrong subs/captions for dubbed videos - Accessibility issue

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#3913]
Wrong subs/captions for dubbed videos - Accessibility issue

I realized this for the first time now since the issue with viewing subs on Windows was corrected... 

**Branch**: develop

**Expected Behavior**: Dubbed videos should have their own transcripts/captions, not recycle the ones used in the English version. 

I tested with:
http://192.168.1.33:8008/learn/khan/math/early-math/cc-early-math-add-sub-basics/cc-early-math-add-sub-intro/subtraction-introduction/
Video/Captions filename is **AO9bHbUdg-M** and it is displayed on both English version of this video, as with the Spanish one. 

**Current Behavior/Steps to reproduce**:

1. Log in as the administrator.
2. Change Language from English to another that you can understand and which has dubbed versions of videos  
3. Download the appropriate language pack and some videos 
4. See if the captions that are displaying make sense with the audio of the video you are playing

**Comment by radinamatic** [comment]
I've tested a few other videos in Spanish, and confirmed what I described previously, namely the same subtitles are used for videos in both languages. However for the:

`/learn/khan/economics-finance-domain/microeconomics/supply-demand-equilibrium/demand-curve-tutorial/price-of-related-products-and-demand/`
(English video filename **-oClpRv7msg.mp4**, Spanish video filename **q8XTsrUVp_4.mp4**, captions/subs filename for both languages **-oClpRv7msg.srt**)

even though their quality leaves a lot to be desired (too many chars, speed too high...), Spanish subs are actually "followable" and do not seem totally "out of place" as in some other videos. This to me suggest the conclusion that the way Spanish version of the videos is made is through mimicking the original version in English as much as possible "to the letter", sometimes with better and sometimes with not so good results in regards to captions/subs.    

Anyhow, this does not seem to be a KA Lite code issue, since it roots in the Khan Academy workflow for dubbing videos, and therefore I'm closing this issue.

**Comment by radinamatic** [comment]
Reopening this as it represents an **accessibility** issue, and should have a proper milestone:

Some people do not use subtitles, but specially those who must use them (deaf or hard on hearing), will have a problem if they are viewing the dubbed version of the video, as the only sub they will see is made as a translation of the original English transcript, and it will not align properly with the dubbed video they are viewing. In fact, those users who do not or cannot rely on the audio content of the video, fare better if they view the **original** video in English, with the translated subs in their own language.

We should consider  adding an optional feature for this user case in the future User Accessibility Settings (#3921), or at the very least an instructions regarding what is the best way for these users to watch KA Lite videos.

**Comment by rtibbles** [comment]
@aronasorman I imagine this is related to the issue we had with the dubbed video mappings being incorrect (dubbed videos were being displayed in the wrong place).

Has that been resolved?

I see not: #4226

**Comment by aronasorman** [comment]
This might be because the KA spreadsheet is unmaintained currently. I'll try making it work next week.

**Comment by rtibbles** [comment]
Oh, I had actually misread @radinamatic's final description of the problem.

> users who do not or cannot rely on the audio content of the video, fare better if they view the original video in English, with the translated subs in their own language.

I am not sure this is fixable within the scope of 0.15. This might be more easily solved with video vectorization, where we can probably hot swap videos more easily?

Either way, I don't see this as fixable in 0.15.

**Comment by radinamatic** [comment]
Opened under "User Accessibility Settings" for Kolibri, and I'm not sure if it can be fixed within the scope of KA Lite. @MCGallaspy @aronasorman

**Comment by radinamatic** [comment]
Opened for discussion on Kolibri repo (and hopefully not needed because of the new schema), so closing here!
